app/templates/buyers/dashboard-team.html

```diff
@@ -0,0 +1,134 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/summary-table.html" as summary %}
+{% block page_title %}Your account - Digital Marketplace{% endblock %}
+
+{% block body_classes %}brief-builder brief-content{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+    {
+        "link": url_for("main.index"),
+        "label": "Home"
+    }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+
+{% with messages = get_flashed_messages(with_categories=true) %}
+    {% if messages %}
+        {% for category, message in messages %}
+            {% if category == 'error' %}
+                <div class="banner-destructive-without-action">
+            {% else %}
+                <div class="banner-success-without-action">
+            {% endif %}
+                    <p class="banner-message">
+            {% if 'requirements_deleted' in message %}
+                    Your requirements &lsquo;{{ message.get('requirements_deleted') }}&rsquo; were deleted
+            {% else %}
+                {{ message }}
+            {% endif %}
+                    </p>
+                </div>
+        {% endfor %}
+    {% endif %}
+{% endwith %}
+
+{% with
+    context = current_user.email_address,
+    heading = current_user.name
+%}
+{% include 'toolkit/page-heading.html' %}
+{% endwith %}
+
+
+{{ summary.heading("Drafts", id="unpublished_requirements") }}
+{% call(item) summary.list_table(
+    draft_briefs,
+    caption="Unpublished requirements caption",
+    empty_message="You have not started any opportunities yet.",
+    field_headings=[
+    "Brief name",
+    "Created",
+    "Unanswered questions",
+    ],
+    field_headings_visible=True
+) %}
+
+    {% call summary.row() %}
+        {{ summary.service_link(item.title, url_for(".view_brief_overview", framework_slug=item.frameworkSlug, lot_slug=item.lot, brief_id=item.id)) }}
+
+        {{ summary.text(item.createdAt|dateformat) }}
+
+        {% if item.unanswered_required > 0 and item.unanswered_optional > 0 %}
+            {{ summary.text("{} required \n{} optional".format(item.unanswered_required, item.unanswered_optional)) }}
+        {% elif item.unanswered_required > 0 %}
+            {{ summary.text("{} required".format(item.unanswered_required)) }}
+        {% elif item.unanswered_optional > 0 %}
+            {{ summary.text("{} optional".format(item.unanswered_optional)) }}
+        {% else %}
+            {{ summary.text() }}
+        {% endif %}
+
+    {% endcall %}
+{% endcall %}
+
+<p><a href="{{url_for('main.info_page_for_starting_a_brief', framework_slug='digital-service-professionals', lot_slug='digital-professionals')}}">Create a new opportunity for a digital specialist</a></p>
```

Comment:
please use DM_FRAMEWORK featureflag to prepare for new framework https://github.com/AusDTO/dto-digitalmarketplace-buyer-frontend/blob/master/app/templates/index.html#L17